src/player.cpp

```diff
@@ -3392,8 +3393,8 @@ int player::print_aim_bars( WINDOW *w, int line_number, item *weapon, Creature *
     // This simplifies the calculation greatly, that's intentional.
     const std::array<std::pair<double, char>, 3> ratings =
         {{ std::make_pair(0.1, '*'), std::make_pair(0.4, '+'), std::make_pair(0.6, '|') }};
-    const std::string confidence_label = _("Confidence: ");
```

Comment:
But eh, I guess it doesn't need to match symbols, since the function of those lines is different. And reducing work for localisation doesn't seem a good reason to prevent a change that makes sense. Colon removal it is.